app/transformers/api/v3/facility_transformer.rb

```diff
@@ -10,8 +10,16 @@ def to_response(facility)
           "teleconsultation_phone_numbers")
         .merge(config: {enable_diabetes_management: facility.enable_diabetes_management,
                         enable_teleconsultation: facility.enable_teleconsultation},
-               sync_region_id: facility.facility_group_id,
+               sync_region_id: sync_region_id(facility),
                protocol_id: facility.protocol.try(:id))
     end
+
+    def sync_region_id(facility)
+      if Flipper.enabled?(:region_level_sync)
+        facility.region.block.id
+      else
+        facility.facility_group.id
```

Comment:
I guess this is inconsistent now 😅  Should the FG be `facility_group.region.id`? So we consistently send back region IDs for everything? I guess we can do that now that our data's all clean.

Reply:
Actually, that might trigger resyncs on _all_ devices. Best to leave as is, and revisit after block-level sync is shipped to all (most) places.

Reply:
I considered this. It shouldn't be `facility_group.region.id` both because it'll trigger a resync and more importantly, the reason why this check is here is for rollback-ability. We want to be able to get back to previous known good state trivially.